components/editor/explainer.tsx

```diff
@@ -115,9 +115,9 @@ const DraggableExplainer = props => {
                         textureSampleLevel(channel1, bilinear, uv, pass, lod)
                     </pre>
                     Keyboard input can be accessed from the provided{' '}
-                    <HiLite>keyDown(keycode: uint)</HiLite> helper function:
+                    <HiLite>keyDown(keycode: u32)</HiLite> helper function:
                     <pre style={{ color: theme.palette.neutral.main }}>
-                        keyDown(32u) // returns true when the spacebar is pressed
+                        keyDown(u32) // returns true when the spacebar is pressed
```

Comment:
This is supposed to be 32 (ASCII for space), though looks like the trailing `u` isn't necessary anymore anyway

Reply:
Ah, yea. Fixed!